packages/integrations/vue/tsconfig.json

```diff
@@ -5,6 +5,7 @@
     "allowJs": true,
     "module": "ES2022",
     "outDir": "./dist",
-    "target": "ES2021"
+    "target": "ES2022",
+    "verbatimModuleSyntax": false
```

Comment:
Q: Why this? (and same in Svelte's)
A: `verbatimModuleSyntax` prevents you from using ESM syntax in CJS files. So it creates an error in the `.cts` file asking it to use CJS syntax. Which is fine, but actually using CJS syntax causes more errors to appear (probably because our codebase assume full ESM). I couldn't figure it out, so I just disabled it.

Hopefully in the future VS Code supports ESM extensions.